docs/modules/operation/pages/alarms/configuring-alarms.adoc

```diff
@@ -51,20 +51,22 @@ NOTE: Any Event that is marked as "donotpersist" in the logmsg element's "dest"
 *NOTE*
 See also: <<events/anatomy-events.adoc#ga-events-anatomy-of-an-event, anatomy of an event>>
 
-.The reduction-key
+== The reduction-key
+
+Alarmd is designed to reduce multiple occurrences of an alarm into a single alarm.
 The critical attribute when defining the alarm-data of an event, is the reduction-key.
 This attribute can contain literal strings as well as references to properties (fields and parameters) of the event.
 The purpose of the reduction-key is to uniquely identify the signature of a problem and, as such, is used to reduce (de-duplicate) events so that only one problem is instantiated.
```

Comment:
```suggestion
The reduction-key uniquely identifies the signature of a problem and, as such, is used to reduce (de-duplicate) events so that only one problem is instantiated.
```